tox.ini

```diff
@@ -7,3 +7,7 @@ skipsdist = true
 deps = -r scripts/nb-tester/requirements.txt
 setenv = PYDEVD_DISABLE_FILE_VALIDATION=1
 commands = python scripts/nb-tester/test-notebook.py {posargs}
+
+[testenv:lint]
+deps = squeaky==0.5.0
+commands = squeaky --check {posargs}
```

Comment:
`--check` won't modify the files, right? That's good. 

If that's true, it'd be good to have a `fmt` environment so that `squeaky` makes changes. Or a more appropriate name could be `fix` since it is making actual changes to source code like removing metadata; it's not only changing how things are formatted.